plugins/nv-filters/nvidia-audiofx-filter.c

```diff
@@ -125,7 +132,7 @@ static void nvidia_audio_destroy(void *data)
 
 	if (ng->nvidia_sdk_dir_found)
 		pthread_mutex_lock(&ng->nvafx_mutex);
-
+	NvAFX_UninitializeLogger();
```

Comment:
Leave a blank line after unenclosed `if` statements, or enclose the `if` with curly braces.

Reply:
done